I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 209, in script
    python:
ScriptError: could not find label 'start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\r.chernitsyn\Downloads\renpy-6.99.8-sdk\renpy\bootstrap.py", line 281, in bootstrap
    renpy.main.main()
  File "C:\Users\r.chernitsyn\Downloads\renpy-6.99.8-sdk\renpy\main.py", line 468, in main
    run(restart)
  File "C:\Users\r.chernitsyn\Downloads\renpy-6.99.8-sdk\renpy\main.py", line 139, in run
    renpy.execution.run_context(True)
  File "C:\Users\r.chernitsyn\Downloads\renpy-6.99.8-sdk\renpy\execution.py", line 714, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 209, in script
    python:
  File "C:\Users\r.chernitsyn\Downloads\renpy-6.99.8-sdk\renpy\script.py", line 843, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start'.

Windows-7-6.1.7601-SP1
Ren'Py 6.99.8.959
